src/libs/antares/study/parameters.cpp

```diff
@@ -626,6 +627,11 @@ static bool SGDIntLoadFamily_Optimization(Parameters& d,
     {
         return stringToGlobalTransmissionCapacities(value, d.transmissionCapacities);
     }
+
+    if (key == "solver-logs")
```

Comment:
We'll need to add it in the documentation too (new section v8.9 in the file format doc).

Reply:
was thinking about it, I thought about it and said to myself that @flomnes  would do it 😛